contrib/idleconn-manpage/genallman.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
You needn't it. The manpage can be generated by txt2man program directly.

Regards,

Eriberto


2016-03-11 15:20 GMT-03:00 Adrian Chadd <notifications@github.com>:

> In contrib/idleconn-manpage/genallman.sh
> <https://github.com/httperf/httperf/pull/39#discussion_r55866678>:
>
> > @@ -0,0 +1,30 @@
> > +#!/bin/bash
>
> Hi,
>
> please rewrite so it's done in bourne shell. People run this on platforms
> that aren't linux/macosx. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/httperf/httperf/pull/39/files#r55866678>.
>